src/gui/src/gui.cpp

```diff
@@ -131,6 +131,7 @@ Selected Gui::makeSelected(std::any object, void* additional_data)
   if (it != descriptors_.end()) {
     return it->second->makeSelected(object, additional_data);
   } else {
+    ord::OpenRoad::openRoad()->getLogger()->warn(utl::GUI, 33, "No descriptor is registered for {}.", object.type().name());
```

Comment:
Using ord::OpenRoad::openRoad() outside the .i file is not intended.   The initGui call is passed the openroad object and the tool should grab what it needs then.  This makes the component more reusable than if it is using global singletons.

I see a fair number of these have crept in so we can clean then in another PR.

Reply:
Sounds good. I'll put mark that down.